src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/fhir/mapper/DeductionTransactionMapper.java

```diff
@@ -0,0 +1,53 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound.fhir.mapper;
+
+import org.hl7.fhir.r4.model.Parameters;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.DeductionDate;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.DeductionReasonCode;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.NewHealthAuthorityName;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.PersonName;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.ReferenceTransactionType;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Transaction;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.fhir.NhsIdentifier;
+import uk.nhs.digital.nhsconnect.lab.results.model.fhir.ParameterNames;
+import uk.nhs.digital.nhsconnect.lab.results.model.fhir.ParametersExtension;
+
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@Component
+public class DeductionTransactionMapper implements FhirTransactionMapper {
+
+    private final static DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+    @Override
+    public Parameters map(Transaction transaction) {
+        var parameters = FhirTransactionMapper.createParameters(transaction);
+        transaction.getPersonName()
+            .map(PersonName::getNhsNumber)
+            .map(NhsIdentifier::new)
+            .ifPresentOrElse(nhsIdentifier -> ParametersExtension.extractPatient(parameters).setIdentifier(List.of(nhsIdentifier)),
+                () -> {throw new EdifactValidationException("Missing mandatory patient NHS number");});
```

Comment:
Please correct the indentation here - there should be a line break after the open curly brace. Check other places in this class too